lib/motor.js

```diff
@@ -597,7 +599,7 @@ util.inherits(Motor, events.EventEmitter);
 Motor.prototype.initialize = function() {
   this.io.pinMode(this.pins.pwm, this.io.MODES.PWM);
 
-  ["dir", "cdir", "brake"].forEach(function(pin) {
+  ["dir", "cdir", "brake", "enable"].forEach(function(pin) {
     if (this.pins[pin]) {
```

Comment:
Yeah, actually now that you mention it: Tessel 2.

I'd change this to `if (typeof this.pins[pin] !== "undefined") {`